typing/typecore.ml

```diff
@@ -3163,7 +3164,8 @@ and type_expect_
             exp_env = env }
       end
   | Pexp_sequence(sexp1, sexp2) ->
-      let exp1 = type_statement env sexp1 in
+      let exp1 = type_statement ~explanation:Sequence_left_hand_side
+          env sexp1 in
```

Comment:
I would have expected a test for `-strict-sequence` to show up in the patch. Where does this happen?

Reply:
It doesn't appear in this patch because it is already present in `type_statement` which was modified previously (loop bodies are handled in exactly the same way as the lhs of a sequence).

Reply:
Thanks!